client.go

```diff
@@ -0,0 +1,102 @@
+package authzed
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"log"
+
+	api "github.com/authzed/authzed-go/arrakisapi/api"
```

Comment:
We should probably put these into a subdirectory of "internal" for now. 

https://docs.google.com/document/d/1e8kOo3r51b2BWtTs_1uADIA5djfXhPT36s6eHVRIvaU/edit#!

Reply:
I don't think we can, because the caller needs access to the raw protos to create requests.